x-pack/legacy/plugins/code/index.ts

```diff
@@ -46,98 +43,29 @@ export const code = (kibana: any) =>
       hacks: ['plugins/code/hacks/toggle_app_link_in_nav'],
     },
     config(Joi: typeof JoiNamespace) {
-      const langSwitches: any = {};
-      LanguageServers.forEach(lang => {
-        langSwitches[lang.name] = Joi.object({
-          enabled: Joi.boolean().default(true),
-        });
-      });
-      LanguageServersDeveloping.forEach(lang => {
-        langSwitches[lang.name] = Joi.object({
-          enabled: Joi.boolean().default(false),
-        });
-      });
       return Joi.object({
+        // Still keep this config item here for the injectDefaultVars
+        // in line 40 here.
         ui: Joi.object({
           enabled: Joi.boolean().default(true),
         }).default(),
         enabled: Joi.boolean().default(true),
```

Comment:
I leave the `enabled` flag in both here and the config in the new place with default value `true`. However, the code app is disabled by default without puting `xpack.code.enabled` in the `kibana.yml`. Any idea on how to fix this? @restrry @joshdover

Reply:
why do we want to have `false` by default?

Reply:
>  the code app is disabled by default 

because it's disabled by default.

Reply:
So this is how it works now. We keep Code enabled on `master` and `7.x`, but disabled on the release branch. Even in the release branch, we set the `xpack.code.ui.enabled` to `false` to disable, but still keep `xpack.code.enabled` to be `true`.

Reply:
> We keep Code enabled on master and 7.x, but disabled on the release branch

how do you achieve that? changing default values in legacy platform config?